### ISSUE TITLE
how to update line with update='append' ?

### ISSUE THREAD

**Issue by zchky** [facebookresearch/visdom#300]
```
    vis_1=viz.line(
            X=np.array([0]),
            Y=np.array([0]),
            name='vis_1',
        win='vis_1',
        )
    viz.line(
            X=np.array([1]),
            Y=np.array([1]),
            win=vis_1,
            name='vis_1',
            update='append',
        )
```

when i update the line with update='append'
it always get separate points.

i want to draw dynamic line which can increase the length automatically

thanks
![newplot 18](https://user-images.githubusercontent.com/17870918/37468872-fb17f758-289e-11e8-89f3-5ee6e86bee8f.png)

**Comment by JackUrb** [comment]
This is related to not being able to plot single points, which is being addressed in #299 right now

**Comment by zchky** [comment]
i mean i want to plot line with every single input like

```
for i in range(100):
    viz.line(
    X=np.array([i+1]),
    Y=np.array([i+1]),
    win=vis_1,
    name='vis_1',
    update='append',
)
```

can i draw a line by using this method?

**Comment by zchky** [comment]
@JackUrb 

Btw, the name of line seem to '1' all the time, i cannot rename it with a new name.

**Comment by JackUrb** [comment]
The line renaming issue is also addressed in #299, which has not landed yet. Please wait for it to be in and then try again 👍

**Comment by JackUrb** [comment]
Closing as #299 has been in for quite some time now
